USER_GUIDE.md

```diff
@@ -368,4 +368,41 @@ The plugin defaults to `appengine-web.xml` based deployment if your project cont
 file. If your project also has an `src/main/appengine/app.yaml` file and you wish to use that, you may temporarily move the
 `appengine-web.xml` file to a different location before deploying.
 
+### How do I deploy an `app.yaml` based thin jar to appengine?
+
+You will probably need to stage the application to include it's dependencies. You might use something like the
```

Comment:
~~it's~~ its

Reply:
I rewrote this whole thing, can you take another look?